src/program.js

```diff
@@ -425,6 +428,11 @@ Example: $0 --help run.
         type: 'boolean',
         default: false,
       },
+      'runAsBinary': {
```

Comment:
Yeah, it was part of some initial attempt that I did, before ending with the changes in the execute function. I just forgot to clean up this.